packages/codebytes/src/index.tsx

```diff
@@ -1,23 +1,61 @@
+import { Box, IconButton } from '@codecademy/gamut';
+import { FaviconIcon } from '@codecademy/gamut-icons';
+import { Background, system } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
 import React, { useState } from 'react';
 
+import { languageOption } from './consts';
 import { Editor } from './editor';
 
 export interface CodeByteEditorProps {
   text: string;
+  language: languageOption;
+  hideCopyButton: boolean;
+  onCopy?: (text: string, language: string) => void;
 }
 
+const EditorContainer = styled(Background)(
+  system.css({
+    border: '1',
+    borderColor: 'gray-900',
+    display: 'flex',
+    flexDirection: 'column',
+    height: '400px',
+    width: '688px',
```

Comment:
Should these values be dynamic?

Reply:
Yes, great catch. I worked on the tues before break with jonathan sanders. I'll push up this change - it was a little more complex than passing in props as we usually do! But yep, totally should be dynamic

Reply:
updated!